[38;5;226m2025-10-18 18:56:09,007 :: WARNING :: root :: config_logging.py :: 26 :: app > region not found. Using default value - global ...[0m
